explorer/interpreter/interpreter.cpp

```diff
@@ -106,6 +107,12 @@ class Interpreter {
                Nonnull<const Value*> destination_type,
                SourceLocation source_loc) -> ErrorOr<Nonnull<const Value*>>;
 
+  // Instantiate a class and its base class(es) from an init struct.
+  auto InstantiateClassWithBase(Nonnull<const StructValue*> init,
```

Comment:
In the interpreter, we use `Instantiate` to mean specifically substituting symbolic names within a value for the corresponding in-scope values of those names, for example "instantiating" the value `Vector(T)` as `Vector(i32)` when `T = i32` is in scope -- somewhat like template instantiation in C++.

This function seems to be doing something unrelated -- more like creating an instance of a class type. Please can you rename this to avoid confusion? Maybe `ConstructClassFromStruct` or `ConvertStructToClass` or something would be clearer? We use the word `Convert` in the interpreter already for this kind of thing.

Reply:
Sure thing, done